sql/mysql/Positive-Technologies/MySqlParser.g4

```diff
@@ -247,7 +247,7 @@ createView
 // details
 
 createDatabaseOption
-    : DEFAULT? (CHARACTER SET | CHARSET) '='? (charsetName | DEFAULT)
+    : DEFAULT? (CHARACTER SET | CHARSET | CHAR SET) '='? (charsetName | DEFAULT)
```

Comment:
Maybe it makes sense to extract these alternative to separated rule since they are used in several places?

```suggestion
    : DEFAULT? charSet '='? (charsetName | DEFAULT)
    
 ...
 
 charSet:
    : CHARACTER SET
    | CHARSET
    | CHAR SET
    ;
```